### ISSUE TITLE
Support waiting for a `scale nodegroup` operation to complete

### ISSUE THREAD

**Issue by cPu1** [weaveworks/eksctl#6396]
eksctl supports a `scale nodegroup` command but the command returns immediately after it has modified the ASG with the new desired size. A `--wait` flag will allow users to wait for the scaling operation to either complete or fail, and monitor any issues related to the scaling. Additionally, a `--timeout` option should also be added to control the maximum duration the command should wait. 

This will also help catch errors like https://github.com/weaveworks/eksctl/issues/6319.

**Comment by Himangini** [comment]
related to #5385

**Comment by TiberiuGC** [comment]
Part of this feature was completed with https://github.com/weaveworks/eksctl/issues/5385. As of now, we have added the wait flag to the command and it has an effect only when scaling up. I.e. waiting for the new nodes to join the cluster.

For now, if scaling down, we log a warning that the flag has no effect. To close this issue, we need to consider what we want to wait on when scaling down (if anything), and open a separate PR for that piece of functionality. Moving back this ticket in refinement to discuss the above.
